src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -116,4 +120,97 @@ public static boolean isValidClassicAddress(String address) {
   public static String toTransactionHash(String transactionBlobHex) {
     return javaScriptUtils.toTransactionHash(transactionBlobHex);
   }
+
+  /**
+   * Convert from units in drops to xrp.
+   *
+   * @param drops An amount of xrp expressed in units of drops.
+   * @return A String representing the drops amount in units of xrp.
+   * @throws Exception if drops is in an invalid format.
+   */
+  public static String dropsToXrp(String drops) throws Exception {
+    String dropsRegex = "^-?[0-9]*['.']?[0-9]*$";
+    Pattern dropsPattern = Pattern.compile(dropsRegex);
+
+    Matcher dropsMatcher = dropsPattern.matcher(drops);
+    if (!dropsMatcher.matches()) {
+      throw new Exception(String.format(
+              "dropsToXrp: invalid value %s, should be a number matching %s.", drops, dropsRegex));
+    } else if (drops == ".") {
```

Comment:
Better to use `".".equals(drops)` instead of `==`.  See https://stackoverflow.com/questions/767372/string-equals-versus